tests/lean/run/simpMatchDiscr.lean

```diff
@@ -1,3 +1,9 @@
+/-!
```

Comment:
I'd suggest to include the discr congr theorems in the test (`#guard_msgs in #print sig`). This is helpful for the next person touching the code to see if they only refactored or actually made a change, and reviewers see the diffs.